packages/frontend/src/filters/number-filter/NumberFilterForm.tsx

```diff
@@ -14,14 +13,11 @@ export const defaultValuesForNewNumberFilter: {
     lessThan: { operator: 'lessThan', value: 0 },
 };
 
-const options: { value: NumberFilter['operator']; label: string }[] = [
-    { value: 'notEquals', label: 'is not equal to' },
-    { value: 'equals', label: 'is equal to' },
-    { value: 'notNull', label: 'is not null' },
-    { value: 'isNull', label: 'is null' },
-    { value: 'lessThan', label: 'is less than' },
-    { value: 'greaterThan', label: 'is greater than' },
-];
+export const defaultValuesForNewBooleanFilter: {
+    [key in BooleanFilter['operator']]: BooleanFilter;
+} = {
+    is: { operator: 'is', value: false },
+};
```

Comment:
Should we move this to the boolean file ?

Reply:
done